Rust/Youtube_Downloader/.gitignore

```diff
@@ -0,0 +1,11 @@
+# Generated by Cargo
+# will have compiled files and executables
+/target/
+
+.*.sw[opn]
+
+test.log
+
+/build/
+
+*.mp[34]
```

Comment:
Put this on the `.gitignore` present in the root directory

Reply:
Resolved and pushed 👍🏾